base/docs/helpdb/Base.jl

```diff
@@ -914,9 +914,9 @@ generate an array of such random numbers.
 randexp
 
 """
-    chop(string)
+    chop(string, i = 1)
```

Comment:
Since `i` tends to refer to an index maybe `n` or `len` would be a better choice for the variable name?